### DISCUSSION TITLE
Upgrade to Expo v51

### DISCUSSION THREAD

**Discussion by samuelwarang** [founded-labs/react-native-reusables#148] [General]
When I try to clone the template and try to set up the application and check using Expo go 
It mentions that the version used in template is 50 and it needs to be 51

**Comment by mrzachnugent** [comment]
Hi @RamchandraWarang9822 

If you pull from `react-native-reusables` and copy the starter-base template again, it now uses expo SDK 51.

If you want to update your current project, I suggest following these steps from expo https://expo.dev/changelog/2024/05-07-sdk-51#%EF%B8%8F-upgrading-your-app
